### PR TITLE
Removed Collapse element. Fixed bunch of tests. Fixed assert.

### PR DESCRIPTION
**Pull request by 886-59-936**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4963575&bid=16404032)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttongroup/VerticalVariationTests.java [hunk 1]
```diff
@@ -74,6 +75,7 @@ public void dropdownMenuTests() {
 
     public void newWindowTitleCheck(String pageTitle) {
         switchToNewWindow();
+        String q = getTitle();
```
Comment (codacy-review) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/buttongroup/VerticalVariationTests.java]: Codacy Issue found: [Avoid unused local variables such as 'q'.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4963575)

File: jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.common.TextTypes;
 import com.epam.jdi.light.elements.complex.ISelector;
+import com.epam.jdi.light.elements.complex.WebList;
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.complex.WebList'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4963575)

**Comment by mf-code** [comment]
[4667] - https://transfer.sh/PZFUS/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4667] - https://transfer.sh/CisRy/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4667] - https://transfer.sh/IPQw5/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4667] - Allure report on Netlify: https://5e3d27ca67a01dc2190bd982--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4672] - Could not save metadata Could not save metadata Could not save metadata

**Comment by RomanIovlev** [comment]
Tests and element already implemented and merged in merge_bootstrap branch
https://github.com/jdi-testing/jdi-light/blob/merge_bootstrap/jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/CollapseTests.java

**Pull request closed without merging**